optuna_integration/xgboost.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any
+
+import optuna
+
+
+use_callback_cls = True
+
+with optuna_integration._imports.try_import() as _imports:
```

Comment:
```sh
optuna_integration/xgboost.py:10: in <module>                                                                                                                                                                                                                                                                           
    with optuna_integration._imports.try_import() as _imports:                                                                                                                                                                                                                                                          
E   NameError: name 'optuna_integration' is not defined 
```

`import optuna_integration` is required before this.

Reply:
Thank you for the review and comment, @y0z. I will get on this.

Reply:
Done! ptal